src/assets/styles/global.scss

```diff
@@ -47,20 +34,39 @@
   --calcite-scrim-background: #{rgba($blk-240, 0.75)};
 }
 
+:root {
+  @extend %type-vars;
+  @include calcite-theme-light-extended();
+
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+
+  --calcite-border-radius: 3px;
+  --calcite-ui-opacity-disabled: 0.5;
+  --calcite-panel-width-multiplier: 1;
+
+  @apply font-sans;
+}
+
+.calcite-theme--dark,
+[theme="dark"] {
+  @include calcite-theme-dark-extended();
+}
+
 .calcite-theme--light,
 [theme="light"] {
-  @include calcite-theme-light();
-  --calcite-theme-name: "light";
-  --calcite-ui-foreground-current: #c7eaff;
-  --calcite-ui-inverse: #{$blk-180};
-  --calcite-ui-inverse-hover: #{$blk-170};
-  --calcite-ui-inverse-press: #{$blk-190};
-  --calcite-alert-dismiss-progress-background: #{rgba($blk-000, 0.8)};
-  --calcite-button-transparent-hover: #{rgba($blk-240, 0.05)};
-  --calcite-button-transparent-press: #{rgba($blk-240, 0.08)};
-  --calcite-input-message-floating-background: #{rgba($blk-000, 0.96)};
-  --calcite-link-blue-underline: #{rgba($h-bb-070, 0.4)};
-  --calcite-scrim-background: #{rgba($blk-000, 0.75)};
+  @include calcite-theme-light-extended();
+}
+
+.calcite-theme--auto {
```

Comment:
We could also call this `calcite-theme--preferred`?

Reply:
`auto` seems descriptive and concise enough to me.